kitsune/customercare/zendesk.py

```diff
@@ -42,6 +50,11 @@ def _user_to_zendesk_user(self, user, email):
             external_id=external_id,
         )
 
+    def get_user_by_email(self, email):
```

Comment:
this should be named to something like `fitler_user_by_email`

Reply:
Ok.